docs/mobile-apps/features/camera-image-injection.md

```diff
@@ -16,6 +16,10 @@ Camera Image Injection, also known as camera mocking, is a Sauce Labs Real Devic
 
 You employ the built-in device camera in your live and automated testing and perform test cases that require taking images with any of the device cameras. To mimic the system camera behavior during a test, you'll provide the app with an image of your choosing. Your app will access the camera and instead of getting back the picture of the device camera, it'll retrieve the uploaded image.
 
+:::caution
+Please make sure you have a debuggable AND not obfuscated version of your application uploaded to Mobile App Storage. 
```

Comment:
Can we provide some guidance for our customers on how to create a debuggable version of the app? Just in a few steps?

Reply:
yup, lets create a new page under mobile app storage and merge it with this:
https://docs.saucelabs.com/mobile-apps/automated-testing/ipa-files/ 

not in this PR though. when the two pages are merged we will add a link to the new
https://docs.saucelabs.com/mobile-apps/ipa-apk/files